mlir/lib/Dialect/Vector/Transforms/VectorInsertExtractStridedSliceRewritePatterns.cpp

```diff
@@ -329,12 +329,76 @@ class DecomposeNDExtractStridedSlice
   }
 };
 
+static void populateFromInt64AttrArray(ArrayAttr arrayAttr,
+                                       SmallVectorImpl<int64_t> &results) {
+  for (auto attr : arrayAttr)
+    results.push_back(llvm::cast<IntegerAttr>(attr).getInt());
+}
```

Comment:
You should be able to use this helper: https://github.com/llvm/llvm-project/blob/7fa0d05a04056aac4365c69c4b515f613a43e454/mlir/lib/Dialect/Utils/IndexingUtils.cpp#L281

Reply:
Thanks!